rest-api-spec/src/main/resources/rest-api-spec/api/create.json

```diff
@@ -1,69 +1,104 @@
 {
-  "create": {
-    "documentation": "http://www.elastic.co/guide/en/elasticsearch/reference/master/docs-index_.html",
-    "methods": ["PUT","POST"],
-    "url": {
-      "path": "/{index}/_create/{id}",
-      "paths": ["/{index}/_create/{id}"],
-      "deprecated_paths" : [
+  "create":{
+    "documentation":{
+      "url":"http://www.elastic.co/guide/en/elasticsearch/reference/master/docs-index_.html",
+      "description":"Creates a new document in the index.\n\nReturns a 409 response when a document with a same ID already exists in the index."
+    },
+    "url":{
+      "paths":[
         {
-          "version" : "7.0.0",
-          "path" : "/{index}/{type}/{id}/_create",
-          "description" : "Specifying types in urls has been deprecated"
-        }
-      ],
-      "parts": {
-        "id": {
-          "type" : "string",
-          "required" : true,
-          "description" : "Document ID"
-        },
-        "index": {
-          "type" : "string",
-          "required" : true,
-          "description" : "The name of the index"
+          "path":"/{index}/_create/{id}",
+          "methods":[
+            "PUT",
+            "POST"
+          ],
+          "parts":{
+            "id":{
+              "type":"string",
+              "required":true,
```

Comment:
I think that we can get rid of the required flag here, now that parts are listed under their own path. All parts are then required in the path that they are part of.

Reply:
Yes, good good catch! The `required` property is indeed redunant in the new format, since effectively _all_ the dynamic parts in a specific path are required.